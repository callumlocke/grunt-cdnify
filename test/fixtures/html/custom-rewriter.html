<html>
  <head>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700">
    <link rel="icon" href="/favicon.ico">
    <link rel="apple-touch-icon" href="/apple-touch-icon.png">
    <link rel="stylesheet" href="../css/custom-rewriter.css">
  </head>

  <body>
  </body>
</html>
