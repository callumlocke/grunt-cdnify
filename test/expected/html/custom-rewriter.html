<html>
  <head>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700">
    <link rel="icon" href="//cdn.example.com/stuff/favicon.ico">
    <link rel="apple-touch-icon" href="//cdn.example.com/stuff/apple-touch-icon.png">
    <link rel="stylesheet" href="//cdn.example.com/stuff/css/custom-rewriter.css">
  </head>

  <body>
  </body>
</html>
